easybuild/tools/toolchain/toolchain.py

```diff
@@ -178,14 +190,18 @@ def __init__(self, name=None, version=None, mns=None, class_constants=None, tcde
         self.mod_full_name = None
         self.mod_short_name = None
         self.init_modpaths = None
-        if self.name != DUMMY_TOOLCHAIN_NAME:
+        if not self.is_system_toolchain():
```

Comment:
Does that really work? don't you need is_system_toolchain(self.name) ?

Reply:
Yeah, method call here :)